src/main/resources/META-INF/resources/webjars/form-flow/0.0.1/js/formFlowHideField.js

```diff
@@ -0,0 +1,13 @@
+function toggleVisibility(checkbox, selector) {
+  let element = document.querySelector(selector);
+  if (element) {
+    element.style.display = checkbox.checked ? 'none' : 'block';
+  }
+}
+
+document.addEventListener("DOMContentLoaded", function () {
+  document.querySelectorAll('input[type="checkbox"][onchange^="toggleVisibility"]').forEach(checkbox => {
+    const fieldSelector = checkbox.getAttribute('onchange').match(/toggleVisibility\(this, '(.+)'\)/)[1];
+    toggleVisibility(checkbox, fieldSelector);
+  });
+});
```

Comment:
I think we can keep this part in il-gcc for now 🤔. It might be a useful for other places in the app, but let's test it out a bit in il-gcc before merging it back to FFL